relay-event-normalization/src/event.rs

```diff
@@ -1519,38 +1506,39 @@ mod tests {
         }
         "#;
 
-        let mut event = Annotated::<Event>::from_json(json).unwrap().0.unwrap();
+        let Annotated(Some(mut event), mut meta) = Annotated::<Event>::from_json(json).unwrap()
+        else {
+            panic!("Invalid transaction json");
+        };
 
-        normalize_platform_and_level(&mut event);
+        normalize_default_attributes(&mut event, &mut meta, &NormalizationConfig::default());
 
-        insta::assert_ron_snapshot!(SerializableAnnotated(&Annotated::new(event)), {}, @r###"
-        {
-          "level": "info",
-          "type": "transaction",
-          "platform": "other",
-        }
-        "###);
+        assert_eq!(event.level.value().unwrap().to_string(), "info");
+        assert_eq!(event.ty.value().unwrap().to_string(), "transaction");
+        assert_eq!(event.platform.value().unwrap().to_owned(), "other");
     }
 
     #[test]
     fn test_normalize_platform_and_level_with_error_event() {
         let json = r#"
         {
-            "type": "error"
+            "type": "error",
+            "exception": {
+                "values": [{"type": "ValueError", "value": "Should not happen"}]
+            }
```

Comment:
`type` is not enough for an event to be considered an error, see 
https://github.com/getsentry/relay/blob/9bf4cbe9a5a2be8ac3d436a666724fd0586d1b17/relay-event-normalization/src/event.rs#L1057